### PR TITLE
JAAS getting started edits

### PR DESCRIPTION
**Pull request by matthewhelmke**
Edits based on review. This first commit is the first round. I'm going to continue by adding a quick "how to destroy a model" under next steps.

### CODE REVIEW

**Comment by matthewhelmke** [comment]
I added one more commit to this. Wanna take one more look @evilnick?

File: src/en/getting-started-jaas.md [hunk 1]
```diff
@@ -118,11 +120,40 @@ Enter the username and set the permissions for the new user.
...
+### Learn the CLI
 
 Most users find the flexibility and power of using JAAS and Juju from the CLI
 rewarding. See [Using JAAS from the command line][jaascli] for more information.
 
+### Destroy a model
+
+When you are done and want to destroy the model you created, click your name.
+
+![juju_jaas_top_panel_name](./media/jaas-username.png)
```
Comment (evilnick) [inline src/en/getting-started-jaas.md]: I really don't like these heavily cropped images. they go against style and look really odd. I don't think it is needed in this case if we are clear - perhaps say "click on your username in the top left" or whtever

File: src/en/getting-started-jaas.md [hunk 2]
```diff
@@ -118,11 +120,40 @@ Enter the username and set the permissions for the new user.
...
+<td align="center" valign="center" border-width="0px" >
+<img src="./media/jaas-model-list.png" alt="description here" />
+<br />
+Click the model name to select the model you want to manage from the list.
+</td>
+
+<td align="center" valign="center" border-width="0px">
+<img src="./media/jaas-destroy-model.png" alt="description here" />
+<br />
+From the options that are revealed, click Destroy model.
```
Comment (evilnick) [inline src/en/getting-started-jaas.md]: backtick `Destroy model`?

File: src/en/getting-started-jaas.md (hunk 1 above)
Comment (matthewhelmke) [inline src/en/getting-started-jaas.md]: Remove the image?
I'm okay with that, just confirming that is what you are saying.

Comment (matthewhelmke) [inline src/en/getting-started-jaas.md]: nvm, did it anyway

File: src/en/getting-started-jaas.md (hunk 2 above)
Comment (matthewhelmke) [inline src/en/getting-started-jaas.md]: good idea, added to all the names of things to be clicked

**Pull request merged**